examples/example_brownian.py

```diff
@@ -0,0 +1,62 @@
+from parcels import Grid, ScipyParticle
+import numpy as np
+from datetime import timedelta as delta
+import math
+import random
+
+
+def two_dim_brownian_flat(particle, grid, time, dt):
+
+    # random.seed() - should a seed be included for reproducibility/testing purposes?
+    # Use equation for particle diffusion.
+    particle.lat += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lat)
+    particle.lon += random.normalvariate(0, 1)*math.sqrt(2*dt*grid.K_lon)
+
+
+def brownian_grid(xdim=200, ydim=200):     # Define a flat grid of zeros, for simplicity.
+    depth = np.zeros(1, dtype=np.float32)
+    time = np.linspace(0., 100000. * 86400., 2, dtype=np.float64)
+
```

Comment:
Both `time` and `depth` are superfluous in this grid. I would simply remove them, and have the function `return Grid.from_data(U, lon, lat, V, lon, lat, mesh='flat')` only, to keep the code as concise as possible